lib/iris/analysis/trajectory.py

```diff
@@ -717,29 +713,31 @@ def __init__(self, src_cube, target_grid_cube):
         This can then be applied to any source data with the same structure as
         the original 'src_cube'.
 
-        Args:
-
-        * src_cube:
+        Parameters
+        ----------
+        src_cube : :class:`~iris.cube.Cube`
             The :class:`~iris.cube.Cube` defining the source grid.
             The X and Y coordinates can have any shape, but must be mapped over
             the same cube dimensions.
-
-        * target_grid_cube:
+        target_grid_cube : :class:`~iris.cube.Cube`
             A :class:`~iris.cube.Cube`, whose X and Y coordinates specify a
             desired target grid.
             The X and Y coordinates must be one-dimensional dimension
             coordinates, mapped to different dimensions.
             All other cube components are ignored.
 
-        Returns:
-            regridder : (object)
+        Returns
+        -------
+        regridder (object)
+            A callable object with the interface::
 
-            A callable object with the interface:
-                `result_cube = regridder(data)`
+                result_cube = regridder(data)
 
             where `data` is a cube with the same grid as the original
             `src_cube`, that is to be regridded to the `target_grid_cube`.
 
+        Notes
+        -----
         .. Note::
```

Comment:
@tkknight Should this be a `.. note::` with a `Notes` ?

Could we drop the rst ?